vendor/src/github.com/docker/libnetwork/drivers/bridge/interface.go

```diff
@@ -52,27 +52,27 @@ func (i *bridgeInterface) exists() bool {
 	return i.Link != nil
 }
 
-// addresses returns a single IPv4 address and all IPv6 addresses for the
+// addresses returns the first IPv4 address, all IPv4 addresses, and all IPv6 addresses for the
 // bridge interface.
-func (i *bridgeInterface) addresses() (netlink.Addr, []netlink.Addr, error) {
+func (i *bridgeInterface) addresses() (netlink.Addr, []netlink.Addr, []netlink.Addr, error) {
```

Comment:
This is pretty unwieldy and I really have no idea what each return actually is without reading the comment above.

Seems like we can at least remove the first return and just call `[0]` to get the "first ipv4 address"